cypress/tests/data/10-ApplicationSetup/50-CreateCategories.cy.js

```diff
@@ -8,7 +8,7 @@
  */
 
 describe('Data suite tests', function() {
-	it('Creates/configures categories', function() {
+	it.skip('Creates/configures categories', function() {
```

Comment:
I guess the test was commented intentionally. I don't have anything against merging without the test, it's just needed to create an issue as a reminder to cover it later.